app/controllers/blocks_controller.rb

```diff
@@ -0,0 +1,49 @@
+class BlocksController < ApplicationController
+  before_action :set_block, only: [:destroy, :edit, :update]
+  respond_to :html
+
+  def index
+    @blocks = current_user.blocks.all.order('title')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.